.gitignore

```diff
@@ -15,13 +16,22 @@ config*.log
 /native/**/*.a
 /native/**/*.so
 /native/**/*.vcxproj.user
+/native/**/*.aps
 /native/**/Win32/
 /native/**/x64/
+/native/**/Release
+/native/exe/HCWIRBlasterCOM/HCWIRBlasterCOM_*
 
+/third_party/ffmpeg/lib*/*_list.c
+/third_party/ffmpeg/lib*/*.ver 
+/third_party/ffmpeg/doc 
 /third_party/**/*.o
 /third_party/**/*.lo
+/third_party/**/*.a
 /third_party/**/*.la
 /third_party/**/*.pc
+/third_party/**/*.def
```

Comment:
Don't add either of these, they refer to existing files in GIT

Reply:
Similar here.  There are some .def intermediate files created for ffmpeg.

Reply:
Yeah, limit it to ffmpeg and it's fine